core/src/main/java/org/infinispan/stats/impl/StatsCollector.java

```diff
@@ -290,6 +299,30 @@ public long getTimeSinceReset() {
       return timeService.timeDuration(resetNanoseconds.get(), TimeUnit.SECONDS);
    }
 
+   @ManagedAttribute(
+         description = "Approximate number of entries currently in the cache, including persisted and expired entries",
+         displayName = "Approximate number of entries"
+   )
+   public long getApproximateEntries() {
+      return CompletionStages.join(persistenceManager.running().approximateSize(AccessMode.BOTH, SEGMENT_0));
```

Comment:
why are you using the `persistenceManager`? `SimpleCache` does not support persistence AFAIK.

Reply:
Yep, you're right